settings.gradle

```diff
@@ -0,0 +1,11 @@
+dependencyResolutionManagement {
+    repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
+    repositories {
+        google()
+        mavenCentral()
+    }
+}
+rootProject.name = "SurveySdk"
```

Comment:
not sure do we need this config?

Reply:
@luongvo it was generated by AS, and as I know, the reason is : 
https://stackoverflow.com/questions/66475730/in-android-studio-arctic-fox-canary-8-the-app-level-build-gradle-does-not-gener

Reply:
@doannimble Nope, I don't think that's the issue here. Put this config in the root build.gradle file and the issue will be resolved.
```
allprojects {
    repositories {
        google()
        mavenCentral()
    }
}
```
We can keep `rootProject.name = "SurveySdk"` config, but the `dependencyResolutionManagement` is quite hacky 🤓

Reply:
@luongvo `Instead of declaring repositories in every subproject of your build or via an allprojects block, Gradle offers a way to declare them in a central place for all project` 
The dependencyResolutionManagement repositories block accepts the same notations as in a project, which includes Maven or Ivy repositories, with or without credentials, etc.

By default, repositories declared by a project will override whatever is declared in settings. You can change this behavior to make sure that you always use the settings repositories.
For your reference : https://docs.gradle.org/current/userguide/dependency_management.html (Centralizing repositories declaration)
This is the new thing with me also, it must be have a good reason why Google included it into the AS template and I'm pretty sure it is not a hack.
What do you think about it?